### ISSUE TITLE
UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.

### ISSUE THREAD

**Issue by RoderickGu** [huggingface/transformers#852]
When I finetune Bert with simple_lm_finetuning.py, there seems an error:
     "UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector."

Will it influence the performance of the finetuning process ? Thanks in advance for any suggestion.

**Comment by thomwolf** [comment]
It should be fine. There are probably your output losses.

**Comment by RoderickGu** [comment]
@thomwolf  Thanks for your reply. Could you explain more about why this happens ? I am still confused though

**Comment by xinsu626** [comment]
I think it may be caused by calculating the loss in the model.

**Comment by calusbr** [comment]
I have the same problem in version 2.2.1, what can it be?

**Comment by RoderickGu** [comment]
In my case, it doesn’t influence the results

**Comment by sociengineer** [comment]
In my case, the speed of training with 4 GPU is the same as 1 GPU. How solve the speed issues?

**Comment by iamlockelightning** [comment]
👀

**Comment by XLechter** [comment]
> In my case, the speed of training with 4 GPU is the same as 1 GPU. How solve the speed issues?

@sociengineer , sorry for bothering you, may I ask if you have solved this problem as I meet the same one. Thanks!

**Comment by sociengineer** [comment]
I couldn't resolve the problem 😭

**Comment by sociengineer** [comment]
> > In my case, the speed of training with 4 GPU is the same as 1 GPU. How solve the speed issues?
> 
> @sociengineer , sorry for bothering you, may I ask if you have solved this problem as I meet the same one. Thanks!

@XLechter I couldn't resolve the problem 😭

**Comment by secain** [comment]
This warning is still there and it's a bummer
